discontinuous-range/index.d.ts

```diff
@@ -0,0 +1,71 @@
+// Type definitions for discontinuous-range 1.0
+// Project: https://www.npmjs.com/package/discontinuous-range
+// Definitions by: Victor Zhou <https://github.com/OiCMudkips>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Represents a discontinuous range of numbers.
+ */
+export declare class DiscontinuousRange {
```

Comment:
These definitions worked for me at runtime, but I was only using these typings to import into another typings file. I switched it up to your style, changed my import in my own typings file to `import DiscontinuousRange = require("discontinuous-range");` and it still worked.